apps/docs/content/guides/platform/backups.mdx

```diff
@@ -135,6 +135,41 @@ A date and time picker will be provided upon pressing the `Start a restore` butt
 
 After locking in the desired point in time to recover to, The Dashboard will then prompt for a review and confirmation before proceeding with the restoration. The project will be inaccessible following this. As such, do ensure to allot for downtime beforehand. This is dependent on the size of the database. The larger it is, the longer the downtime will be. Once the confirmation has been given, the latest physical backup available is downloaded to the project and the database is partially restored. WAL files generated after this physical backup up to the specified point-in-time are then downloaded. The underlying records of transactions in these files are replayed against the database to complete the restoration. The Dashboard will display a notification once the restoration completes.
 
+## Restore to a new project
+
+Supabase provides a convenient way to restore data from an existing project into a completely new one. Whether you're using physical backups or Point-in-Time recovery (PITR), this feature allows you to duplicate projects with ease, perform safe testing, or recover data for analysis. This feature is exclusive to users on paid plans and requires that physical backups are enabled for the source project.
+
+<Admonition type="note">
+
+PITR is an additional add-on available for organizations on a paid plan with physical backups enabled.
+
+</Admonition>
+
+To begin, switch to the source project—the project containing the data you wish to restore—and go to the [database backups](https://supabase.com/project/_/database/backups/restore-to-new-project) page. Select the **Restore to a New Project** tab.
```

Comment:
```suggestion
To begin, switch to the source project—the project containing the data you wish to restore—and go to the [database backups](https://supabase.com/dashboard/project/_/database/backups/restore-to-new-project) page. Select the **Restore to a New Project** tab.
```

Reply:
@Crispy1975 i don't see this tab in prod Dashboard yet. are we waiting to release it?

Reply:
I'll double check the settings to be sure all is as it sold be.

Reply:
Done.